src/api/createProductTeam/index.py

```diff
@@ -1,38 +1,22 @@
+from event.api_step_chain import execute_step_chain
 from event.environment import BaseEnvironment
-from event.event_processing.steps import event_processing_steps
 from event.logging.logger import setup_logger
-from event.logging.step_decorators import logging_step_decorators
-from event.response.steps import response_steps
-from event.step_chain import StepChain
-from event.versioning.steps import get_steps_for_requested_version, versioning_steps
+
+from .src.v1.steps import steps as v1_steps
 
 
 class Environment(BaseEnvironment):
     DYNAMODB_TABLE: str
 
 
 cache = {**Environment.build().dict()}
-step_decorators = [*logging_step_decorators]
-pre_steps = [*versioning_steps, *event_processing_steps]
-post_steps = [*response_steps]
+versioned_steps = {"1": v1_steps}
```

Comment:
Just a thought for keeping things safe, this versioned_steps as a constant in the class doesnt enforce this structure, might be worth having a class that requires a version and steps to ensure its a forced pattern?